source/get-started.txt

```diff
@@ -0,0 +1,48 @@
+.. _pymongo-get-started:
+
+===========
+Get Started
+===========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: tutorial
+ 
+.. meta::
+   :description: Learn how to create an app to connect to MongoDB deployment by using the PyMongo driver.
+   :keywords: quick start, tutorial, basics
+
+.. toctree::
+
+   /get-started/download-and-install/
+   /get-started/create-a-deployment/
+   /get-started/create-a-connection-string/
+   /get-started/connect-to-mongodb/
+   /get-started/next-steps/
+
+Overview
+--------
+
+This guide shows you how to create an application that uses
+{+driver-short+} to connect to a MongoDB cluster hosted on MongoDB Atlas. If
+you prefer to connect to MongoDB using a different driver or programming
+language, see our :driver:`list of official drivers <>`.
+
+{+driver-short+} is a library of functions that you can use to connect
+to and communicate with MongoDB.
```

Comment:
Q: Is it more than too (like classes)? I honestly don't know

Reply:
Good point. I think it's safe to just remove "of functions" and keep it more general. Does that work?

Reply:
How about "package"?

https://stackoverflow.com/questions/19198166/whats-the-difference-between-a-module-and-a-library-in-python

Reply:
I personally like the way it is introduced in the first sentence of the [current documentation](https://pymongo.readthedocs.io/en/stable/#overview), with the caveat that 'package' is preferable to 'distribution', but I think that it is important to say Python. 

"PyMongo is a Python distribution. containing tools for working with [MongoDB](http://www.mongodb.org/), and is the recommended way to work with MongoDB from Python."